src/System.Device.Gpio/System/Device/Pwm/PwmChannel.cs

```diff
@@ -48,6 +55,7 @@ protected virtual void Dispose(bool disposing)
         /// <param name="frequency">The frequency in hertz.</param>
         /// <param name="dutyCyclePercentage">The duty cycle percentage represented as a value between 0.0 and 1.0.</param>
         /// <returns>A PWM channel running on Windows 10 IoT.</returns>
+        [Obsolete("Use Board.CreatePwmChannel instead")]
```

Comment:
Same comment than with SpiDevice. I think with an introduction of a feature like this, we shouldn't force people down that path every time and instead keep allowing both ways for creating objects so people that don't want to go through the Board abstraction are not force d to.